admin-cli/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "scripts": {
     "build": "echo \"Compiling admin-cli\" && (cd ../shared && node build.js --install) && tsc",
-    "mla:build": "yarn run build && yarn run mla",
+    "mla:build": "npm run build && npm run mla",
```

Comment:
Could gone with `npx yarn build && npx yarn mla` instead, however, we seem to use `npm run ...` in all other npm scripts as well, so I rather have this consistent.

Reply:
I'm ok with that but want to raise awareness that we *do have some cases* where the usage of yarn over npm is crucial. Not this one though.

Reply:
Yes, those cases are mainly `install` commands, which we will run through `npx yarn` everywhere.